### ISSUE TITLE
feedback button overlaps content on zoom

### ISSUE THREAD

**Issue by vellip** [liqd/a4-meinberlin#518]
Part of the BITV-Test: "1.4.4a Schriftgröße variabel" and "1.4.4b Bei Zoom auf 200% benutzbar"
----
It's not really a big thing, but the test fails because of it.

**Comment by xi** [comment]
It also overlaps on small screen?

**Comment by vellip** [comment]
When I tested it did not, it disappeared there.
